packages/amplify_api_rest/android/build.gradle

```diff
@@ -0,0 +1,51 @@
+group 'com.amazonaws.amplify.amplify_api_rest'
+version '1.0-SNAPSHOT'
+
+buildscript {
+    ext.kotlin_version = '1.3.50'
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.5.0'
```

Comment:
Latest stable as of today is [4.1.1](https://mvnrepository.com/artifact/com.android.tools.build/gradle/4.1.1)

Reply:
Thanks made a note to look into updating that in our merged branch.